spec/rails_helper.rb

```diff
@@ -38,14 +40,22 @@
   abort e.to_s.strip
 end
 RSpec.configure do |config|
+  config.include Warden::Test::Helpers
+  config.after :each do
+    Warden.test_reset!
+  end
+
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
-  config.fixture_path = "#{::Rails.root}/spec/fixtures"
+  # config.fixture_path = "#{::Rails.root}/spec/fixtures"
 
   # If you're not using ActiveRecord, or you'd prefer not to run each of your
   # examples within a transaction, remove the following line or assign false
   # instead of true.
   config.use_transactional_fixtures = true
 
+  config.include Rails.application.routes.url_helpers
+  config.include Capybara::DSL
+
```

Comment:
if you setup `Capybara.current_driver = :selenium_chrome` here, you will be able to remove it from your specs :)

Reply:
Not really. As I mentioned in some comment - all Capybara tests worked before rebase. After rebase they pass only if I place the driver in every and each describe.

Reply:
Based on [that](https://github.com/teamcapybara/capybara#drivers) maybe setting up a `default_driver` will help?..

Reply:
default_driver helped, but still I would like to get back to the point, where Capybara tests passed without opening a browser and executing the sequence.

Reply:
Ha, that(tests passing without opening browser) means they were basing on the default driver. Default driver, according to [capybara docs](https://github.com/teamcapybara/capybara#selecting-the-driver) is `:rack_test` - maybe try setting up this one? 🤔

Reply:
Also, idk how the `coverage/index.html` is with us again, but it needs to `git rm` go :D

Reply:
:rack_test didn't help ;(,    index.html removed

Reply:
what if `:rack_test` didn't help because you kept all of the `Capybara.current_driver = :selenium_chrome` in the rest of specs? Or have you remembered to clean these up?

Reply:
Nah, I removed current_driver from spec files. They are no longer there